sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClientBuilder.java

```diff
@@ -151,40 +206,51 @@ public ContainerRegistryClientBuilder retryPolicy(RetryPolicy retryPolicy) {
 
 
     /**
-     * Sets the service version that will be targeted by the client.
-     * If the version is not set the latest version is used by default.
+     * Sets the {@link ContainerRegistryServiceVersion} that is used when making API requests.
+     * <p>
+     * If a service version is not provided, the service version that will be used will be the latest known service
+     * version based on the version of the client library being used. If no service version is specified, updating to a
+     * newer version the client library will have the result of potentially moving to a newer service version.
```

Comment:
```suggestion
     * version based on the version of the client library being used and when updating to a
     * newer version of the client library may result in moving to a newer service version.
```

Merged the sentences as both began with the same idea of not setting a service version.